src/main/java/mate/controller/DisplayAllCarsController.java

```diff
@@ -0,0 +1,26 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class DisplayAllCarsController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("mate");
+    private final CarService carsService =
+            (CarService) injector.getInstance(CarService.class);
+    private final DriverService driverService =
+            (DriverService) injector.getInstance(DriverService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.setAttribute("cars", carsService.getAll());
+        req.setAttribute("drivers", driverService.getAll());
```

Comment:
Why do we need all drivers in DisplayAllCarsController?

Reply:
As I am newbie in webapps and JSP, I was very curious as to how I can put a list of drivers next to the car. It was like a challenge for me :) And I think, that I done it. ))

Reply:
![image](https://user-images.githubusercontent.com/89127223/181652722-c0fab598-97ee-4751-aafb-52652b95bc07.png)